Tests/Marketplace/configure_and_install_packs.py

```diff
@@ -82,27 +85,28 @@ def xsoar_configure_and_install_flow(options, branch_name: str, build_number: st
     if not content_path:
         raise Exception('Could not find content path')
 
-    # all packs that should be installed
+    # Create a list of all packs that should be installed
     packs_to_install = set(Build.fetch_pack_ids_to_install(options.pack_ids_to_install))
-    logging.info(f'packs to install before filtering by minServerVersion {packs_to_install}')
+    logging.info('Packs to install before filtering by minServerVersion:' + ', '.join(packs_to_install))
```

Comment:
why you changed it from f'string to join?

Reply:
Just to have it printed in a more readable manner.
(`minServerVersion: X, Y, Z` instead of `minServerVersion {'X', 'Y', 'Z'}`)

There is however a missing space there
```suggestion
    logging.info('Packs to install before filtering by minServerVersion: ' + ', '.join(packs_to_install))
```

If you think the previous print is better, let me know and I'll revert that.

Reply:
If you want to print it like this, I would do it this way:
```suggestion
    logging.info(f'Packs to install before filtering by minServerVersion: {", ".join(packs_to_install)}')
```
But I don't think it is so meaningful that you need to change all the logs to be like this and make them more clumsy

Reply:
I'll revert that then